Heisenbug in test suite

**Describe the bug**
This test occasionally passes and occasionally fails: https://github.com/open-zaak/open-zaak/blob/master/src/openzaak/components/catalogi/tests/admin/test_import_export_zaaktype.py#L666

Example build:
https://github.com/open-zaak/open-zaak/pull/808/checks?check_run_id=1687643142

Failing test:
```
======================================================================
FAIL: test_simultaneous_zaaktype_imports (openzaak.components.catalogi.tests.admin.test_import_export_zaaktype.ZaakTypeAdminImportExportTests)
----------------------------------------------------------------------

Traceback (most recent call last):
  File "/home/runner/work/open-zaak/open-zaak/src/openzaak/components/catalogi/tests/admin/test_import_export_zaaktype.py", line 744, in test_simultaneous_zaaktype_imports
    self.assertEqual(zaaktype1.zaaktype_omschrijving, "zaaktype1")

AssertionError: 'zaaktype2' != 'zaaktype1'
```

**To Reproduce**
Running CI many times

**Expected behavior**
The test should always pass or always fail

--- a/src/openzaak/components/catalogi/tests/admin/test_import_export_zaaktype.py
+++ b/src/openzaak/components/catalogi/tests/admin/test_import_export_zaaktype.py
@@ -739,7 +739,7 @@ class ZaakTypeAdminImportExportTests(MockSelectielijst, WebTest):
         response2 = response2.form.submit("_select")
 
         self.assertEqual(ZaakType.objects.count(), 2)
-        zaaktype1, zaaktype2 = ZaakType.objects.all()
+        zaaktype1, zaaktype2 = ZaakType.objects.all().order_by("zaaktype_omschrijving")
 
         self.assertEqual(zaaktype1.zaaktype_omschrijving, "zaaktype1")
         self.assertEqual(zaaktype2.zaaktype_omschrijving, "zaaktype2")

